code/modules/reagents/chemistry/reagents.dm

```diff
@@ -154,8 +154,10 @@
  *
  */
 /datum/reagent/proc/on_mob_life(mob/living/carbon/affected_mob, seconds_per_tick, times_fired)
-	SHOULD_CALL_PARENT(TRUE)
```

Comment:
Even removing the code, I think it's still a good idea to have ``SHOULD_CALL_PARENT`` in case something is added in the future.